### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/buildah/commit/e8d3725b778010f955e2d8194aa9ea4b350b3445#commitcomment-45299927

#### How to verify it

The action reports that the changes in this PR would make it happy: 
https://github.com/jsoref/buildah/commit/a13702773644b5ece0933ba35c3a3cd6a1d892ad

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

While I can squash at any time, it's really best for people to thumbs up/down the individual changes first and let me squash last.

If there are files you want me to skip (changelogs are a common example), just let me know.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Changelog
 
 ## v1.18.0 (2020-11-16)
-    Fix testing error caused by simultanious merge
+    Fix testing error caused by simultaneous merge
```
Comment (rhatdan) [inline CHANGELOG.md:6]: Please do not change the changelog file.  Sadly this needs to match the actual Changelog in git.

File: changelog.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 - Changelog for v1.18.0 (2020-11-16)
-  * Fix testing error caused by simultanious merge
+  * Fix testing error caused by simultaneous merge
```
Comment (rhatdan) [inline changelog.txt:2]: Do not change this file either for the same reason.

**Review by rhatdan — APPROVED** [review]
Other then not changing changlog.* this looks good.

Please squash your commits.

**Comment by rhatdan** [comment]
Thanks @jsoref

**Comment by rhatdan** [comment]
LGTM
@TomSweeneyRedHat @nalind @ashley-cui @QiWang19 @umohnani8 PTAL
@containers/podman-maintainers PTAL

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ Once you have decided to contribute to Buildah by working on an issue, check our
 backlog of [open issues](https://github.com/containers/buildah/issues) looking
 for any that do not have an "In Progress" label attached to it.  Often issues
 will be assigned to someone, to be worked on at a later time.  If you have the
-time to work on the issue now add yourself as an assignee, and set the
+time to work on the issue now add yourselves as an assignee, and set the
```
Comment (ashley-cui) [inline CONTRIBUTING.md:40]:
```suggestion
time to work on the issue now, add yourself as an assignee, and set the
```
Grammar nit: 
"you" is singular so "yourself" should also be singular. But we do need a comma after "now"

**Review by ashley-cui — COMMENTED** [review]
One small small grammar nit, but otherwise LGTM

**Pull request merged**